json-schema-provider/src/main/java/io/confluent/kafka/schemaregistry/json/JsonSchema.java

```diff
@@ -165,27 +165,37 @@ public class JsonSchema implements ParsedSchema {
 
   // prepopulate the draft 2019-09 metaschemas, as the json-sKema library
   // only prepopulates the draft 2020-12 metaschemas
-  private static final Map<URI, String> prepopulatedMetaSchemas = ImmutableMap.of(
-      URI.create("https://json-schema.org/draft/2019-09/schema"),
-      readFromClassPath("/metaschemas/draft/2019-09/schema"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/core"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/core"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/validation"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/validation"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/applicator"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/applicator"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/meta-data"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/meta-data"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/format"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/format"),
-      URI.create("https://json-schema.org/draft/2019-09/meta/content"),
-      readFromClassPath("/metaschemas/draft/2019-09/meta/content"),
-      // The following are to account for URIs with trailing #
-      URI.create("https://json-schema.org/draft/2019-09/schema#"),
-      readFromClassPath("/metaschemas/draft/2019-09/schema"),
-      URI.create("https://json-schema.org/draft/2020-12/schema#"),
-      readFromClassPath("/metaschemas/draft/2020-12/schema")
-  );
+  private static final Map<URI, String> prepopulatedMetaSchemas = 
```

Comment:
Replace Immutable.of with ImmutableMap.Builder since the former can only handle 10 pairs